src/main/java/racingcar/domain/RacingFacilitator.java

```diff
@@ -0,0 +1,74 @@
+package racingcar.domain;
+
+import racingcar.dto.Record;
+import racingcar.dto.StageRecord;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class RacingFacilitator {
+
+    public static final String WINNER_NAME_DELIMITER = ",";
+    private final PositiveNumber racingTryCount;
+    private final Racing racing;
+
+    private final List<StageRecord> records = new ArrayList<>();
+
+    public List<StageRecord> processRacing() {
+        for (int stage = 0; stage < racingTryCount.getNumber(); stage++) {
+            List<Record> records = racing.start();
+            writeStageRecord(stage, records);
+        }
+
+        return this.records;
+    }
+
+    private void writeStageRecord(int stage, List<Record> records) {
+        this.records.add(new StageRecord(stage, records));
+    }
+
+    public String getWinners() {
+        StageRecord finalStageRecord = records
+                .stream()
+                .reduce((first, second) -> second)
+                .orElse(null);
+
+        if (finalStageRecord == null) {
+            throw new IllegalStateException("최종 stage record가 없습니다.");
+        }
+
+        List<Record> finalRank = finalStageRecord.getRank();
+        Record highRecord = finalStageRecord.highRecord();
+
+        return finalRank
+                .stream()
+                .filter(it -> Objects.equals(it.getMoveCount(), highRecord.getMoveCount()))
```

Comment:
getter로 각각의 도메인의 내부값을 꺼내 연산에 사용하고 있어요. 이는 객체지향적이지 않습니다. 
객체에게 질의를 하고 응답을 받는 식의 구조로 만드는게 좋습니다. 
```suggestion
                .filter(it -> highRecord.equalsPosition(it))
```

이런식으로 메세지를 주고받도록 해보면 좋을 것 같네요.